src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,29 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, hidden = false) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = hidden;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(allmostDead) {
```

Comment:
`almostDead` is not the best name for this variable. 
Method bite gets the name of the animal, let's write smt like `animalName` or `target`